src/content/docs/en/guides/astro-db.mdx

```diff
@@ -351,27 +351,55 @@ Your table schema will change overtime as your project grows. You can safely tes
 
 When [creating a Studio project from the dashboard](#astro-studio), you will have the option to create a GitHub CI action. This will automatically migrate schema changes when merging with your repository's main branch.
 
-You can also push schema changes via the CLI using the `astro db push` command:
+You can also push schema changes via the CLI using the `astro db push --remote` command:
 
 <PackageManagerTabs>
   <Fragment slot="npm">
   ```sh
-  npm run astro db push
+  npm run astro db push --remote
   ```
   </Fragment>
   <Fragment slot="pnpm">
   ```sh
-  pnpm astro db push
+  pnpm astro db push --remote
   ```
   </Fragment>
   <Fragment slot="yarn">
   ```sh
-  yarn astro db push
+  yarn astro db push --remote
   ```
   </Fragment>
 </PackageManagerTabs>
 
-This command will verify changes can be made without data loss and guide on recommended schema changes to resolve conflicts. If a breaking schema change _must_ be made, add the `--force-reset` flag to reset all production data.
+This command will verify changes can be made without data loss and guide on recommended schema changes to resolve conflicts.
+
+:::note
+If a breaking schema change _must_ be made, add the `--force-reset` flag to reset all production data. __This will destroy your database__, so only do so if you do not have important user data already.
+:::
+
+<StudioHeading>
+### Renaming tables
+</StudioHeading>
+
+If you have already pushed your schema to Astro Studio and want to rename a table you can take one of thes approaches:
+
+If you *do __not__ have production data* then you can reset your database using the `--force-reset` flag. This flag will drop all of the tables in the database and create new ones so that it matches your current schema exactly.
+
+If you __do__ have production data you need to be more careful. You can perform a rename with the following steps:
+
+1. Keep the __old table__ schema in your config and add the `deprecated: true` property:
+
+    ```ts
+    const Ingredient = defineTable({
+      deprecated: true
+      // ...
+    });
+    ```
+2. Add the __new table__ schema alongside the old table.
+3. Push to Astro Studio with `astro db push --remote`. This will add the new table and mark the old as deprecated.
+4. __Important__: Update any of your code to use the new table instead of the old table. You might need to migrate data to the new table as well.
+5. Once you are confident at the old table is no longer used you can remove it. Remove the schema from your `config.ts`.
+6. Push to Astro Studio with `astro db push --remote`. The old table will be dropped, leaving only the new, renamed table.
```

Comment:
```suggestion
It is possible to rename a table after pushing your schema to Astro Studio.

If you **do not have any important production data**, then you can [reset your database](#pushing-breaking-schema-changes) using the `--force-reset` flag. This flag will drop all of the tables in the database and create new ones so that it matches your current schema exactly.

To rename a table while preserving your production data, you must perform a series of non-breaking changes to push your local schema to Astro studio safely.

The following example renames a table from `Comment` to `Feedback`:

1. In your database config file, add the `deprecated: true` property to the table you want to rename:

    ```ts title="db/config.ts" ins={2}
    const Comment = defineTable({
      deprecated: true
    	columns: {
    		author: column.text(),
    		body: column.text(),
  		}
    });
    ```
2. Add a new table schema (matching the existing table's properties exactly) with the new name:

	  ```ts title="db/config.ts" ins={9-14}
    const Comment = defineTable({
      deprecated: true
    	columns: {
    		author: column.text(),
    		body: column.text(),
  		}
   	});

		const Feedback = defineTable({
			  columns: {
    			author: column.text(),
    			body: column.text(),
  		}
		});
    ```
3. [Push to Astro Studio](#pushing-table-schemas) with `astro db push --remote`. This will add the new table and mark the old as deprecated.
5. Update any of your local project code to use the new table instead of the old table. You might need to migrate data to the new table as well.
6. Once you are confident at the old table is no longer used in your project, you can remove the schema from your `config.ts`:
		```ts title="db/config.ts" del={1-7}
    const Comment = defineTable({
      deprecated: true
    	columns: {
    		author: column.text(),
    		body: column.text(),
  		}
   	});

		const Feedback = defineTable({
			  columns: {
    			author: column.text(),
    			body: column.text(),
  		}
		});
    ```
7. Push to Astro Studio again with `astro db push --remote`. The old table will be dropped, leaving only the new, renamed table.
```
I may need to fiddle with syntax/formatting here, and we'll add our new `<Steps />` component afterward, but this is what I'm thinking here.